lib/Doctrine/ORM/Cache/AssociationCacheEntry.php

```diff
@@ -9,30 +9,14 @@
  */
 class AssociationCacheEntry implements CacheEntry
```

Comment:
You've removed the DocBlock of `CollectionCacheEntry`, I think you can remove this one too.